### ISSUE TITLE
More flexible scaling options

### ISSUE THREAD

**Issue by mrleblanc101** [antfu-collective/icones#337]
### Clear and concise description of the problem

Currently, I find that the "large" option is still a bit small, it's hard to following when searching through the 13 000+ materials icons.

### Suggested solution

Instead of small and large option, maybe we could get a "+/-" option, so we can scale however big/small we like ?

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by antfu** [comment]
I guess we could have a slider, PR welcome if you want to work on that!
